src/components/button-toggle/button-toggle.style.ts

```diff
@@ -126,6 +128,31 @@ const StyledButtonToggle = styled.button<StyledButtonToggleProps>`
       }
       cursor: not-allowed;
     `};
+
+  ${({ variant }) =>
```

Comment:
**thought**: this all looks good, and just testing the storybook it looks fine, but i'll leave the specific checking of these tokens against the DS to QA